xbmc/video/GUIViewStateVideo.cpp

```diff
@@ -436,6 +436,26 @@ bool CGUIViewStateWindowVideoNav::AutoPlayNextItem()
   return CSettings::Get().GetBool("videoplayer.autoplaynextitem");
 }
 
+bool CGUIViewStateWindowVideoNav::JumpToFirstUnplayedItem()
+{
+  if (m_items.IsVideoDb())
+  {
+    NODE_TYPE NodeType = CVideoDatabaseDirectory::GetDirectoryChildType(m_items.GetPath());
+    switch (NodeType)
+    {
+    case NODE_TYPE_EPISODES:
+    case NODE_TYPE_SEASONS:
+      return CSettings::Get().GetBool("videolibrary.jumptofirstunplayeditem");
```

Comment:
I guess you need to add an additional sort method check in here. Jumping to first unplayed when sort method is eg. rating makes no sense and will confuse the users.